### ISSUE TITLE
What is the purpose of the code below ?

### ISSUE THREAD

**Issue by mehmetcanbalci-Notrino** [zekageri/ESP32_I2S-ADC-DMA-Plotting-to-WEB#1]
As i can see I2S_EVENT_RX_DONE is a enum , so it is not a variable . Why did you put in if ?

```
if(I2S_EVENT_RX_DONE){
     Plotting(i2s_read_buff);
   }
```

**Comment by zekageri** [comment]
Nah, nothing. That was just a test sketch. And it is not working properly. I abandoned it because i can not use the ESP32's ADC properly. There is no solution for continous adcing. Not even with i2s. :(

**Comment by lakshminarayanaks79** [comment]
Hi Zekageri I need some help with i2s  could you be able to answer me i am doing a project to read an ECG signal and sen it via Bluetooth but I need to Send the raw value from the sensor to DMA via i2s my signal is analog can it be possible to do it bcoz I didnt find a way to integrate ADC with DMA.

please do help me with my question.

Thanks in advance
